api/include/opentelemetry/trace/tracer-registry.h

```diff
@@ -0,0 +1,28 @@
+#ifndef INCLUDE_OPEN_TELEMETRY_TRACE_TRACER_REGISTRY_H_
+#define INCLUDE_OPEN_TELEMETRY_TRACE_TRACER_REGISTRY_H_
```

Comment:
We've decided to [go with `#pragma once`](https://github.com/open-telemetry/opentelemetry-cpp/pull/9#discussion_r337296570).

https://github.com/open-telemetry/opentelemetry-cpp/blob/c85b20b46dcf9a73268a9d7b8b7646b4d917e450/api/include/opentelemetry/nostd/string_view.h#L1